modules/obj/test/obj-loader.spec.js

```diff
@@ -27,3 +27,27 @@ test('OBJLoader#parseText', t => {
 
   t.end();
 });
+
+test('OBJLoader#parseTextAsync', t => {
+  if (typeof Worker === 'undefined') {
+    t.comment('Worker is not usable in non-browser environments');
+    t.end();
+    return;
+  }
+
+  const objBinary = new TextEncoder().encode(OBJ_ASCII);
```

Comment:
Side note: Feels like this should be done automatically in the loader drivers. The text/binary handling is still quite fragile there.